third_party/repositories.bzl

```diff
@@ -312,8 +309,7 @@ module CYaml {
             namespaced_swift_library(
                 name = "Yams",
                 srcs = ["Sources/Yams/*.swift"],
-                deps = [":CYaml"],
-                cc_libs = [":CYamlLib"],
+                deps = [":CYaml", ":CYamlLib"],
```

Comment:
FYI because of rules_swift changes you can use this rule now to eliminate one of these:


```python
load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")

cc_library(
    name = "CYaml",
    srcs = glob([
        "Sources/CYaml/src/*.c",
        "Sources/CYaml/src/*.h",
    ]),
    hdrs = glob([
        "Sources/CYaml/include/*.h",
    ]),
    copts = ["-fPIC"],
    includes = ["Sources/CYaml/include"],
    linkstatic = True,
)

swift_library(
    name = "Yams",
    srcs = glob(["Sources/Yams/*.swift"]),
    defines = ["SWIFT_PACKAGE"],
    visibility = ["//visibility:public"],
    deps = [":CYaml"],
)
```

Reply:
Thanks! This certainly helps me clean up some stuff in here

Reply:
I was also able to remove the module map we created before as well! Updated now